### PR TITLE
style: downloads copy and style changes - LESQ-405

### PR DESCRIPTION
**Pull request by JBR90**
## Description

**I want copy and colour updates on the downloads page**

https://www.notion.so/oaknationalacademy/I-want-copy-and-colour-updates-on-the-downloads-page-6d8a7d430d8c44f8b6232614db504b15

https://www.figma.com/file/LSjbzdvHo17XqXGBRNJjQP/Old-%2F-new-content?type=design&node-id=3898%3A312974&mode=design&t=yxKxja11QL4N1mTH-1

## Acceptance Criteria:

- [x]  Grey background on page
- [x]  Remove the copy above the input fields to reduce the repetition and redundant text
    - [x]  ****Find your school in the field below (required)****
    - [x]  ****For regular updates from Oak (optional)****
- [x]  Change shaded box behinds t’s and c’s acceptance from blue to grey
- [x]  Update the newsletter sign-up copy under the email box
- [x]  Remove the X/X files selected
- [x]  Add page heading “Download” (h1 in HTML) but styled as Heading4 (desktop and tablet) and Heading 5 on mobile

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2048--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-foundation/units/surds/lessons/accuracy-of-final-answers/downloads?preselected=slide+deck

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
@JBR90 Wow this is coming along so quickly. Small changes:
- [ ] Mobile: the whole content should have 16px margin on left and right, currently it stretches all the way to the edges
- [ ] “Your details’ and ‘Lesson resources” heading should drop to Heading 6 on mobile, sorry forgot. To add that to Acs
- [x] Desktop margin under “your details” can we increase it to 32px, looks a bit tight with the yellow label underneath it

File: src/styles/theme/default.theme.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ const theme: OakTheme = {
     oakGrey4: "#575756",
     oakGrey5: "#2D2D2D",
     oakGrey6: "#1B1B1B",
+    oakGrey30: "#E4E4E4",
```
Comment (k-huggs) [inline src/styles/theme/default.theme.ts:27]: should these new colours be in the //oak new colours section below?

**Review by k-huggs — APPROVED** [review]
LGTM ✅

File: src/styles/theme/default.theme.ts (hunk 1 above)
Comment (JBR90) [reply src/styles/theme/default.theme.ts:27]: I'm not sure, @Monika-Koziol are these new design kit colors ?

**Review by rightsaidjames — APPROVED** [review]
Looks good, I've checked all of the design changes in DevTools to confirm they're all there.

**Pull request merged**